pkg/kubelet/kubelet_node_status_test.go

```diff
@@ -1214,12 +1218,15 @@ func TestTryRegisterWithApiServer(t *testing.T) {
 }
 
 func TestUpdateNewNodeStatusTooLargeReservation(t *testing.T) {
```

Comment:
Can you add one more test case with node-status-max-images=0?

Reply:
added in TestUpdateNewNodeStatus